modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/MvPartitionMeta.java

```diff
@@ -26,20 +27,50 @@
 public class MvPartitionMeta extends PrimitivePartitionMeta {
     private final byte[] groupConfig;
 
+    private final byte[] snapshotInfo;
+
     /** Constructor. */
     public MvPartitionMeta(
             long lastAppliedIndex,
             long lastAppliedTerm,
             byte[] groupConfig,
-            @Nullable LeaseInfo leaseInfo
+            @Nullable LeaseInfo leaseInfo,
+            byte[] snapshotInfo
     ) {
         super(lastAppliedIndex, lastAppliedTerm, leaseInfo);
 
         this.groupConfig = groupConfig;
+        this.snapshotInfo = snapshotInfo;
```

Comment:
I just wanted to make sure, do you think there will be no problems with backward compatibility, say, when starting on a new version, if there is `null` here, then this is not a problem, what do you think?

Reply:
In this PR, this class is created on-the-fly, it is not persisted anywhere, so it can't be null. Also, this PR is only needed for the colocation case, which is disabled on older versions